plugin/symbols/symbols.talon

```diff
@@ -12,19 +12,19 @@ empty escaped (dub string | dub quotes): user.insert_between('\\"', '\\"')
 empty string: user.insert_between("'", "'")
 empty escaped string: user.insert_between("\\'", "\\'")
 (inside parens | args): user.insert_between("(", ")")
-inside (squares | square brackets | list): user.insert_between("[", "]")
-inside (bracket | braces): user.insert_between("{", "}")
+inside (squares | brackets | list): user.insert_between("[", "]")
+inside braces: user.insert_between("{", "}")
 inside percent: user.insert_between("%", "%")
 inside (quotes | string): user.insert_between("'", "'")
 inside (double quotes | dub quotes): user.insert_between('"', '"')
 inside (graves | back ticks): user.insert_between("`", "`")
 angle that:
     text = edit.selected_text()
     user.paste("<{text}>")
-(square | square bracket) that:
+(square | bracket) that:
```

Comment:
Same — keep "square bracket" here.

Reply:
Done in https://github.com/talonhub/community/pull/1344/commits/f0dbeae39e4f65a7b00fb070efa896af4f805cc0.